gato/policy/gato_policy.py

```diff
@@ -89,6 +82,11 @@ def __init__(
                 config=config,
             )
             embed_dim = config.n_embd
+            #self.embed_token = self.transformer.wte
+            assert self.transformer.wte.weight.shape[0] == self.text_tokens, "pretrained token/expected mimsatch" # potentially make text_tokens dynamic
+            # expand embedding dictionary up to vocab_size
+            self.embed_token = nn.Embedding(self.vocab_size, embed_dim)
+            self.embed_token.weight.data[:self.text_tokens] = self.transformer.wte.weight.data
```

Comment:
I updated this to expand the pretrained wte

Reply:
Perhaps I didn't encounter this since I was always using text_prop as 1.0 and doing an isolated testing. This makes a lot of sense. Thanks for the addition!